File: 4107-find-missing-elements/find-missing-elements.go
import "slices"

func findMissingElements(nums []int) []int {
	n := len(nums)
	slices.Sort(nums)
	ans := []int{}

	for i := nums[0]; i < nums[n-1]; i++ {
		if !slices.Contains(nums, i) {
			ans = append(ans, i)
		}
	}

	return ans
}

File: 4107-find-missing-elements/find-missing-elements.py
class Solution:
    def findMissingElements(self, nums: List[int]) -> List[int]:
        mn = min(nums)
        mx = max(nums)
        ans = []

        for i in range(mn, mx + 1):
            if i not in nums:
                ans.append(i)

        return ans
